study.md

```diff
@@ -112,26 +112,27 @@ var myModule = (function(arg, transform) {
 1.  Leave `f` as-is or alter it to create a more interesting module.
 1.  Leave the statements toward the end of the document as-is or alter them to
     do something more interesting.
-1.  Run the script with `node index.js`.
+1.  Run the script with `node inline.js`.
 
 ### Checkpoint 1
 
 1.  Open [`lib/inline.js`](lib/inline.js).
 1.  Put a pair of parens (`(` and `)`) around our declaration of `f`.
+    Assign this function to a new variable, and update the assignment of
+    `ourModule` to equal this new variable.
```

Comment:
Agreed. That was my original proposed change, I didn't intend to include it in the commit.